glustercli/cmd/utils.go

```diff
@@ -57,6 +58,35 @@ func sizeToMb(value string) (uint64, error) {
 	return size, nil
 }
 
+// logn is used to find the unit size the given MB belongs to.
+// 1024 MB will return 1
+// 1048576 Mb will return 2 and so on
+func logn(n, b float64) float64 {
+	return math.Log(n) / math.Log(b)
+}
+
+// HumanReadable converts the MB given into respective unit which is readable
```

Comment:
humanReadable converts size given in MB into a more human readable unit

Reply:
done